lib/CurrencyExchangeRateFields/CurrencyExchangeRateFields.js

```diff
@@ -28,6 +28,7 @@ const CurrencyExchangeRateFields = ({
   initialCurrency,
   isCurrencyRequired,
   isNonInteractive,
+  isSetExchangeRateNonIntaractive,
```

Comment:
it looks like you're trying to implement custom logic in a common place. Why the behavior is different for orders and invoice?

Reply:
In Orders, there is one case when `currency` and `exchangeRate` become non-interactive, in Invoices - one for - approved invoice and another for paid invoice.